crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -69,14 +73,26 @@ where
         self.committee_nodes_with_stake.clone()
     }
 
-    #[cfg(not(feature = "randomized-leader-election"))]
+    #[cfg(not(any(
+        feature = "randomized-leader-election",
+        feature = "fixed-leader-election"
+    )))]
     /// Index the vector of public keys with the current view number
     fn get_leader(&self, view_number: TYPES::Time) -> PUBKEY {
         let index = usize::try_from(*view_number % self.nodes_with_stake.len() as u64).unwrap();
         let res = self.nodes_with_stake[index].clone();
         TYPES::SignatureKey::get_public_key(&res)
     }
 
+    #[cfg(feature = "fixed-leader-election")]
+    /// Only get leader in fixed set
+    /// Index the fixed vector (first fixed_leader_for_gpuvid element) of public keys with the current view number
+    fn get_leader(&self, view_number: TYPES::Time) -> PUBKEY {
+        let index = usize::try_from(*view_number % self.fixed_leader_for_gpuvid as u64).unwrap();
```

Comment:
General question since I'm a bit unfamiliar: are we (for now) expecting the fixed leaders to be the first n nodes? Is it moderately easy to guarantee we can have the first n nodes be the GPU ones? Maybe it is - we can probably just have them connect to the orchestrator first 😃

Reply:
Yes! We're expecting the fixed leaders to be the first n nodes. Yes that's exactly my plan: we connect n validators to the orchestrator first so they'll get lower index, then start other validators. The feature [fixed-leader] is just to enable this function of choosing leaders from first n index-lower node.